Doc/library/webbrowser.rst

```diff
@@ -143,9 +143,9 @@ for the controller classes, all defined in this module.
 +------------------------+-----------------------------------------+-------+
 | ``'windows-default'``  | :class:`WindowsDefault`                 | \(2)  |
 +------------------------+-----------------------------------------+-------+
-| ``'macosx'``           | :class:`MacOSX('default')`              | \(3)  |
+| ``'macosx'``           | :class:`MacOSXOSAScript('default')`     | \(3)  |
 +------------------------+-----------------------------------------+-------+
-| ``'safari'``           | :class:`MacOSX('safari')`               | \(3)  |
+| ``'safari'``           | :class:`MacOSXOSAScript('safari')`      | \(3)  |
```

Comment:
This section should be backported

Reply:
I don't think we can reasonably call renaming MacOSX to MacOSXOSAScript a "backport" of this PR since it is about the deprecation. I'll of course accept PRs to 3.10 and 3.9 that make this doc change.